app/src/main/java/app/revanced/integrations/youtube/patches/announcements/AnnouncementsPatch.java

```diff
@@ -35,6 +37,12 @@ public static void showAnnouncement(final Activity context) {
         // Check if there is internet connection
         if (!Utils.isNetworkConnected()) return;
 
+        // If GmsCore is not installed, the https call will always fail with a security exception.
+        if (GmsCoreSupport.gmsIsNotInstalled()) {
```

Comment:
This makes the patch dependent on GmsCore support patch. Whats the reason for security exceptions?